src/late_multicellular_stage/LateMulticellularSpecies.cs

```diff
@@ -190,6 +194,23 @@ private static float CalculateBrainPowerFromLayout(MetaballLayout<MulticellularM
         return result;
     }
 
+    private static float CalculateMuscularPowerFromLayout(MetaballLayout<MulticellularMetaball> layout, float scale)
+    {
+        float result = 0;
+
+        foreach (var metaball in layout)
+        {
+            if (metaball.CellType.IsMuscularTissueType())
+            {
+                // TODO: check that volume scaling in physically sensible way (using GetVolume) is what we want here
```

Comment:
This comment is copy-pasted without adjustment from the brain power calculation...